src/scm/Model.ts

```diff
@@ -477,6 +479,12 @@ export class Model implements Disposable {
 
                 const chnum: number = parseInt(num.toString());
 
+                const prefix = config.get<string>('ignoredChangelistPrefix');
+                if (prefix && description.startsWith(prefix)) {
```

Comment:
I wonder what is happening with the default `perforce.ignoreChangelistPrefix` value.
Would it ignore changelist with empty description, by default?

Reply:
It shouldn't be a problem as empty string is evaluated to false and second part of condition won't be evaluated. There will be a problem when someone decide to use 0 as prefix ;)